bodhi/server/templates/update.html

```diff
@@ -791,16 +803,17 @@ <h3 class="nomargin">Notes about this update:</h3>
 
         </div>
       </div>
+      <div class="hidden-md hidden-lg">
+         <div class="row">
+            <div class="col-md-12">
+               <button type="submit" class="pull-right btn btn-default">
+                  <span class="indicator fa fa-comment" data-spinclass="indicator fa fa-spinner fa-spin"></span>
+                  Add Comment
+                </button>
+              </div>
+            </div>
+          </div>
```

Comment:
I will be searching for it also, but can it be merged untill its new solution is found :)